### PR TITLE
Add colour to hf mfdes command

### PR DESCRIPTION
**Pull request by dunderhay**

### CODE REVIEW

File: client/cmdhfmfdes.c [hunk 1]
```diff
@@ -168,27 +168,27 @@ void getKeySettings(uint8_t *aid) {
...
 
-        str = (resp.data.asBytes[3] & (1 << 3)) ? "YES" : "NO";
-        PrintAndLogEx(NORMAL, "   [0x08] Configuration changeable       : %s", str);
-        str = (resp.data.asBytes[3] & (1 << 2)) ? "NO" : "YES";
-        PrintAndLogEx(NORMAL, "   [0x04] CMK required for create/delete : %s", str);
-        str = (resp.data.asBytes[3] & (1 << 1)) ? "NO" : "YES";
-        PrintAndLogEx(NORMAL, "   [0x02] Directory list access with CMK : %s", str);
-        str = (resp.data.asBytes[3] & (1 << 0)) ? "YES" : "NO";
-        PrintAndLogEx(NORMAL, "   [0x01] CMK is changeable              : %s", str);
+        str = (resp.data.asBytes[3] & (1 << 3)) ? _GREEN_("YES") : _RED_("NO");
```
Comment (iceman1001) [inline client/cmdhfmfdes.c]: not need RED for no here.  
Here its important to figure out which settings is advantages for us... ie use green. 
if not,  just normal white.

File: client/cmdhfmfdes.c [hunk 2]
```diff
@@ -198,13 +198,13 @@ void getKeySettings(uint8_t *aid) {
...
-            PrintAndLogEx(WARNING, "   Can't read key-version");
+            PrintAndLogEx(WARNING, _RED_("   Can't read key-version"));
             return;
         }
-        PrintAndLogEx(NORMAL, "");
-        PrintAndLogEx(NORMAL, "   Max number of keys       : %d", resp.data.asBytes[4]);
-        PrintAndLogEx(NORMAL, "   Master key Version       : %d (0x%02x)", resp.data.asBytes[3], resp.data.asBytes[3]);
-        PrintAndLogEx(NORMAL, "   ----------------------------------------------------------");
+        PrintAndLogEx(SUCCESS, "");
+        PrintAndLogEx(SUCCESS, "   Max number of keys       : %d", resp.data.asBytes[4]);
```
Comment (iceman1001) [inline client/cmdhfmfdes.c]: yellow?

Comment (dunderhay) [reply client/cmdhfmfdes.c]: Changes in 4a8a0ec

**Comment by iceman1001** [comment]
Excellent !

**Pull request merged**